src/Products.jl

```diff
@@ -165,7 +165,8 @@ function locate(lp::LibraryProduct; verbose::Bool = false,
                 if platforms_match(platform, platform_key_abi())
                     if isolate
                         # Isolated dlopen is a lot slower, but safer
-                        if success(`$(Base.julia_cmd()) -e "import Libdl; Libdl.dlopen(\"$dl_path\")"`)
+                        dl_esc_path = replace(dl_path, "\\"=>"\\\\")
```

Comment:
Ooh, we gotta be careful with this one; I don't think this will work on every platform.  What is more likely to work is to `replace(dl_path, "\\"=>"/")`, since forward-slash (even though it's not the default returned by tools on Windows) should work everywhere.

Reply:
I would not expect that any other platform than windows will have a`\` in the path.
Moreover, I just copied the idea of [line 395](https://github.com/JuliaPackaging/BinaryProvider.jl/blob/master/src/Products.jl#L395)

Reply:
Just checked whether your proposal would work on windows, and interestingly it does pass the test. But ImageMagick does not build with `"\\"=>"/"` whereas it does with `"\\" => "\\\\"`.
So I would encourage someone with a Linux machine to also test whether ImageMagick builds successfully with my version.